CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+cmake_minimum_required(VERSION 3.23)
+project(acquire-common)
+cmake_policy(SET CMP0079 NEW) # use targets from other directories
+enable_testing()
+
+include(cmake/aq_require.cmake)
+include(cmake/git-versioning.cmake)
+include(cmake/ide.cmake)
+include(cmake/msvc.cmake)
+include(cmake/simd.cmake)
+
+set(CMAKE_POSITION_INDEPENDENT_CODE ON)
+
+set(CMAKE_C_STANDARD 11)
+set(CMAKE_CXX_STANDARD 20)
+
+aq_require(acquire-core-libs)
```

Comment:
After this reorg, do we still need `aq_require` or can we just use `add_subdirectory` directly now?